lib/pushmi_pullyu/preservation_queue.rb

```diff
@@ -61,11 +61,11 @@ def next_item
   end
 
   def wait_next_item
-    loop do
-      element = next_item
-      return element if element.present?
-      sleep @poll_interval
-    end
+    # loop do
```

Comment:
Is a second loop needed @mbarnett ? Or can just using the main loop work fine here? Will remove comments if you agree if this is okay.